environment-approvals.yml

```diff
@@ -108,6 +108,8 @@ prod:
   - repo: https://github.com/hmcts/rd-shared-infrastructure.git
   - repo: https://github.com/hmcts/rd-judicial-api.git
   - repo: https://github.com/hmcts/rd-judicial-data-load.git
+  - repo: https://github.com/hmcts/rd-location-ref-api.git
```

Comment:
link to OAT or ATO please

Reply:
This is standard release and CHG5002780 is change request. This is being agreed with release team,IA & other stake holders.That OAT and ATO will be part final release when Fee & Pay consumer go live with us in near future.Please approve this PR.